data/World/Water Temple MQ.json

```diff
@@ -65,7 +65,7 @@
         "dungeon": "Water Temple",
         "locations": {
             "Water Temple MQ Freestanding Key": "
-                Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump or can_bonk_crate",
+                ((Hover_Boots or can_use(Scarecrow) or logic_water_north_basement_ledge_jump) and can_bonk_crate)",
```

Comment:
Being overly pedantic but: outer set of parenthesis are unnecessary here.

Reply:
Updated